src/calculateRentalCost.js

```diff
@@ -1,10 +1,25 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const basePrice = days * pricePerDay;
+
+  if (days >= longTerm) {
+    return basePrice - longTermDiscount;
+  } else if (days >= shortTerm) {
+    return basePrice - shortTermDiscount;
+  }
```

Comment:
The 'else' keyword is unnecessary after 'return' as the function exits after the 'return' statement. Removing the 'else' statement will simplify the code flow and adhere to best practices.